src/index.js

```diff
@@ -518,12 +520,12 @@ class TargetCoreAPI {
   }
 
   /** Delete Offer.
-  * Deletes the content offer referenced by the provided id.
-  *
-  * @param id {integer} Offer id.
-  * @param options {Object} sdk options
-  * @param options.headers headers to pass to API call
-  */
+   * Deletes the content offer referenced by the provided id.
+   *
+   * @param id {integer} Offer id.
```

Comment:
if changing one @param, I think all @params should be the same format (type then name) so it is consistent. This occurs in all JSDocs